src/components/dropdown/section-dropdown.tsx

```diff
@@ -122,6 +129,8 @@ const SectionDropdownComponent = <T extends unknown>({
         onPress={() => {
           scroll();
 
+          trackEvent(testID, AnalyticsEventCategory.General, testIDProperties);
```

Comment:
`develop` branch already contains `TouchableWithAnalytics` component for this purpose